odbc/src/api/col_attr_describe_tests.rs

```diff
@@ -484,8 +484,8 @@ mod unit {
             (Desc::SQL_DESC_FIXED_PREC_SCALE, 0),
             (Desc::SQL_DESC_LENGTH, 0),
             (Desc::SQL_DESC_NULLABLE, 0),
-            (Desc::SQL_DESC_OCTET_LENGTH, 0),
-            (Desc::SQL_DESC_PRECISION, 0),
+            (Desc::SQL_DESC_OCTET_LENGTH, 65535),
```

Comment:
https://learn.microsoft.com/en-us/sql/odbc/reference/appendixes/transfer-octet-length?view=sql-server-ver16

OCTENT LENGTH is max size. We choose 2^16 to be sort of reasonable. If this is 0, direct query does not work for several conversions to VARCHAR/STRING

Reply:
https://learn.microsoft.com/en-us/sql/odbc/reference/appendixes/column-size-decimal-digits-transfer-octet-length-and-display-size?view=sql-server-ver16

```Similarly, the values for transfer octet length do not come from SQL_DESC_LENGTH. They come from the SQL_DESC_OCTET_LENGTH of a field of a descriptor for all character and binary types. There is no descriptor field that holds this information for other types.```

Reply:
Explained in other thread where you asked the same question

Reply:
The ODBC documentation is terrible.

https://www.ibm.com/docs/en/db2-for-zos/11?topic=attributes-precision-sql-data-types

Reply:
Right, if you look at what ibm did in db2 they said precision for char types to be length. I'm comfortable doing what they do since the ODBC spec seems to be silent. (this link https://www.ibm.com/docs/en/db2-for-zos/11?topic=attributes-precision-sql-data-types)

Reply:
I think the issue here is that simple types needs to set precision to MAX_STRING_SIZE, also, when it's not a fixed size (like ObjectID)

Reply:
I made that change and am also leaving OCTENT_LENGTH as 0. I tested to make sure that didn't break Direct Query. Direct Query is relying only on PRECISION for strings, thankfully.